README.md

```diff
@@ -175,6 +177,35 @@ npx nx test (lib_name)
 npx nx test --test-match=/data/dev/gn/ui/libs/common/src/lib/services/bootstrap.service.spec.ts
 ```
 
+#### End-to-end-tests
+
+You can test the datahub app by page :
+
+- home page
+- search page
+- organisations page
+- dataset pages
+
+##### To run the tests with the interface :
+
+Start docker from 'support-services', and then in the 'geonetwork-ui' folder :
+
+```shell script
+ng e2e appname --watch
```

Comment:
```suggestion
nx e2e appname --watch
```

Reply:
Error while running this command.
BTW, it's better to also have the command to run in the PR description, it's easier to review :pray: 
```
 >  NX   Cannot find configuration for task datahub:e2e
```

Reply:
Well, I get `nx command not found` by running `nx e2e appname --watch`. Works with `ng` only for me which is why I commited that command.

Reply:
`nx` and `ng` are the same command, it's weird that you don't have `nx`, but I've installed it globally.
Your could then use `npx nx`, ng should be removed soon (from next nx migration).

With `ng e2e datahub --watch` I have the same error, is `datahub` a correct input ?

Reply:
No, the right input is `datahub-e2e`. Let me know if that was the issue.

Reply:
Generally we should invite users to use the `nx` cli; to run it, you probably need to do `npx nx e2e datahub-e2e --watch` for example (npx is needed if the binary is not globally installed, which is probably the case)